tests/functional/test_trace_config.py

```diff
@@ -0,0 +1,75 @@
+#
+# Copyright(c) 2020 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause-Clear
+#
+
+from core.test_run import TestRun
+from test_tools.fs_utils import remove, read_file
+from utils.iotrace import IotracePlugin, parse_json
+
+custom_repo_path = "/var/lib/octf/custom_trace"
+config_file = "/etc/octf/octf.conf"
+
+
+def test_get_repo_path():
+    """
+        title: Test of iotrace -C -G functionality.
+        description: |
+          Verify the output of 'iotrace -M -G' is proper.
+        pass_criteria:
+          - Getting path to trace repository returns proper path.
+    """
+    with TestRun.step("Get trace repository path from OCTF config file."):
+        default_trace_repo_path = parse_json(read_file(config_file))[0]["paths"]["trace"]
+
+    with TestRun.step("Get trace repository path by command."):
+        trace_repo_path = IotracePlugin.get_trace_repository_path()
+        shortcut_trace_repo_path = IotracePlugin.get_trace_repository_path(True)
+
+    with TestRun.step("Compare with default path"):
+        if trace_repo_path != default_trace_repo_path:
+            TestRun.LOGGER.error(f'Command output should be {default_trace_repo_path}\n'
+                                 f'but its actually {trace_repo_path}')
+
+        if shortcut_trace_repo_path != default_trace_repo_path:
+            TestRun.LOGGER.error(f'Shortened command output should be {default_trace_repo_path}\n'
+                                 f'but its actually {shortcut_trace_repo_path}')
+
+
+def test_set_repo_path():
```

Comment:
I think what's needed is listing traces after path changed. Of course you need to generate io traces under both repo path.